contracts/multisig/src/workerset.rs

```diff
@@ -0,0 +1,50 @@
+use std::collections::BTreeSet;
+
+use crate::key::PublicKey;
+use crate::msg::Signer;
+use axelar_wasm_std::Participant;
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{HexBinary, StdError, Uint256};
+use sha3::{Digest, Keccak256};
+
+#[cw_serde]
+pub struct WorkerSet {
+    pub signers: BTreeSet<Signer>,
+    pub threshold: Uint256,
+    // for hash uniqueness. The same exact worker set could be in use at two different times,
+    // and we need to be able to distinguish between the two
+    pub created_at: u64,
```

Comment:
How can this be used to differentiate worker sets that are created in the same block?

Reply:
I didn't think of that, thank you! Just made a commit that addresses it and have a few questions below.

1. I added a `Timestamp` field for the time it was created at, called `created_at_time`. Does this make `created_at_block` redundant? I feel like having block information is more "native" and is still worth keeping.

2. Adding the `created_at_time` field broke the test case `encoding::abi::test::test_new_command_batch_with_operator_transfer` for me. While I was looking into how to fix it, I noticed the data it's checking if it's equal to is a predefined value stored in `multisig-prover/src/test/test_data.rs` that is returned in the function `encoded_data_with_operator_transfer()` (hex value is too long to copy and paste into this comment).
How did they come up with that hex output? I assume I'll need to create a new hex value to update the test case with to accomodate the new `created_at_time` field, and want to learn how to create such a value myself without just replacing the current value with the new outputted value.

Reply:
@cjcobb23 I think my questions would be more relevant to you. Let me know what you think!

Reply:
to be resolved in future PR